Bottom: 59c376c697def8496ef9c517b72403d48f835bc0
Top:    f3d9f59d8d6fc669aa82fa77cf80e0331a645bb1
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-07 18:27:11 +0000

Refresh of debugging-on-again

---

diff --git a/src/polygon1.c b/src/polygon1.c
index cb3590e..23c9cf2 100644
--- a/src/polygon1.c
+++ b/src/polygon1.c
@@ -39,6 +39,7 @@
       are marked
 */
 
+#undef NDEBUG
 #include	<assert.h>
 #include	<stdlib.h>
 #include	<stdio.h>
